stacks/core/README.md

```diff
@@ -13,16 +13,30 @@ If you don't have a domain or can't configure the main DNS server, you can confi
 
 On Windows, the hosts file is usually located under `c:\Windows\System32\Drivers\etc\hosts`. On Unix systems, it is commonly found at `/etc/hosts`. Regardless of you operating system, administrator or super user privileges will be required.
 
-### Deploy: stage 1
+### Deploy
 
-Stage 1 contains the core services. The next sections provide the instructions to configure and deploy them. Whenever a configuration file is mentioned, it refers to a file found at `stack/env.d`.
+The `core` stack uses multiple compose files to setup the services and reach the desired state. Definitions common to both development and production settings are grouped in the `base` directory and specific ones are grouped in the `dev/prod` directories. For example, when configuring `traefik` as service's load balancer, you can deploy its development definition to avoid exposing services without need. Also, it can be necessary to deploy the services into multiple machines, which could be done by using the `multi-node` definitions.
 
-#### Deploy
+Deploy the services with multiple compose files:
 
-Deploy the stage 1 services:
+```bash
+docker stack deploy --compose-file prod/traefik.yml -c base/base.yml knot-cloud-core
```

Comment:
This command doesn't work if used in the `stacks` directory (created by `knot-cloud init core <path>`). Considering the previous step and that this documentation (in the previous iteration) was worked around the `init` command, I suggest we either stick to this approach (removing `prod/` and `base/`, since these directories don't exist) and then, in another example, specify how a developer should do this for production. 

A better approach, as discussed with @netoax, is to not document this around the `init` command as well as to introduce the concept of [workspaces](https://vtex.io/docs/concepts/workspace/) (or similar), given that target developers for this tool will most likely already have cloned repositories. This helps to seamlessly move between developing environments, without having to jump around duplicated directories or repos for `dev`/`prod` environments.

Also, I suggest we always use the entire flag :point_down: 
```suggestion
docker stack deploy --compose-file prod/traefik.yml --compose-file base/base.yml knot-cloud-core
```

Reply:
In order to solve this, another example was added. Let's work on the improvements we discussed before to the official release. For now, I think this PR is enough and made good progress towards it.